js/src/modal.js

```diff
@@ -458,65 +458,48 @@ class Modal extends BaseComponent {
-      SelectorEngine.find(SELECTOR_STICKY_CONTENT)
-        .forEach(element => {
-          const actualMargin = element.style.marginRight
-          const calculatedMargin = window.getComputedStyle(element)['margin-right']
-          Manipulator.setDataAttribute(element, 'margin-right', actualMargin)
-          element.style.marginRight = `${Number.parseFloat(calculatedMargin) - this._scrollbarWidth}px`
-        })
-
+      this._setElement(SELECTOR_STICKY_CONTENT, 'marginRight', calculatedValue => calculatedValue - this._scrollbarWidth)
       // Adjust body padding
-      const actualPadding = document.body.style.paddingRight
-      const calculatedPadding = window.getComputedStyle(document.body)['padding-right']
-
-      Manipulator.setDataAttribute(document.body, 'padding-right', actualPadding)
-      document.body.style.paddingRight = `${Number.parseFloat(calculatedPadding) + this._scrollbarWidth}px`
+      this._setElement('body', 'paddingRight', calculatedValue => calculatedValue + this._scrollbarWidth)
     }
 
     document.body.classList.add(CLASS_NAME_OPEN)
   }
 
-  _resetScrollbar() {
-    // Restore fixed content padding
-    SelectorEngine.find(SELECTOR_FIXED_CONTENT)
+  _setElement(selector, styleProp, callback) {
+    SelectorEngine.find(selector)
       .forEach(element => {
-        const padding = Manipulator.getDataAttribute(element, 'padding-right')
-        if (typeof padding !== 'undefined') {
-          Manipulator.removeDataAttribute(element, 'padding-right')
-          element.style.paddingRight = padding
-        }
+        const actualValue = element.style[styleProp]
+        const calculatedValue = window.getComputedStyle(element)[styleProp]
+        Manipulator.setDataAttribute(element, styleProp, actualValue)
+        element.style[styleProp] = callback(Number.parseFloat(calculatedValue)) + 'px'
       })
+  }
 
-    // Restore sticky content and navbar-toggler margin
-    SelectorEngine.find(`${SELECTOR_STICKY_CONTENT}`)
-      .forEach(element => {
-        const margin = Manipulator.getDataAttribute(element, 'margin-right')
-        if (typeof margin !== 'undefined') {
-          Manipulator.removeDataAttribute(element, 'margin-right')
-          element.style.marginRight = margin
-        }
-      })
+  _resetScrollbar() {
+    // Reset changed values
 
+    // Restore fixed content padding
+    this._resetElement(SELECTOR_FIXED_CONTENT, 'paddingRight')
+    // Restore sticky content and navbar-toggler margin
+    this._resetElement(SELECTOR_STICKY_CONTENT, 'marginRight')
     // Restore body padding
-    const padding = Manipulator.getDataAttribute(document.body, 'padding-right')
-    if (typeof padding === 'undefined') {
-      document.body.style.paddingRight = ''
-    } else {
-      Manipulator.removeDataAttribute(document.body, 'padding-right')
-      document.body.style.paddingRight = padding
-    }
+    this._resetElement('body', 'paddingRight')
+  }
+
+  _resetElement(selector, styleProp) {
```

Comment:
Better if you name these methods like: `_resetElementAttributes` and `_setElementAttributes`